tests/data/datacom.json

```diff
@@ -23,7 +23,7 @@
                 {
                     "sysName": "",
                     "sysObjectID": ".1.3.6.1.4.1.3709",
-                    "sysDescr": null,
+                    "sysDescr": "",
```

Comment:
should we make empty strings null?

Reply:
Probably should be null.  LibreNMS seems inconsistent with sys* variables and null/""